Monika After Story/game/script-topics.rpy

```diff
@@ -3709,10 +3710,70 @@ label monika_penname:
+                                m 1n "...I mean, I won't question your choice of pen names, but..."
+                                m "If you wanted to name yourself after a character in this game, you should have picked me!"
+                                $ persistent._mas_penname = penname
+                                $ penbool = True
+                            elif lowerpen =="natsuki":
+                                m 2c "..."
+                                m 1n "Well, I guess I shouldn't assume that you named yourself after {i}our{/i} Natsuki."
+                                m "It's something of a common name." 
+                                m "You might make me jealous, though."
+                                $ persistent._mas_penname = penname
+                                $ penbool = True
+                            elif lowerpen == "yuri":
+                                m 2c "..."
+                                m 1n "Well, I guess I shouldn't assume that you named yourself after {i}our{/i} Yuri."
+                                m "It's something of a common name."
+                                m 1i "Of course, there's something else that name could refer too..."
+                                if persistent.gender =="F":
+                                  m 5 "And well...I could get behind that, since it's you~"
+                                $ persistent._mas_penname = penname
+                                $ penbool = True                                
+                            elif lowerpen =="monika":
+                                m "..."
+                                m 1 ebla "Aww, did you pick that for me?" 
+                                m "Even if you didn't, that's so sweet!"
+                                $ persistent._mas_penname = penname
+                                $ penbool = True
+                            elif not lowerpen:
+                                m 1j "Well, go on! You can type 'nevermind' if you've chickened out~"
+                            elif lowerpen =="nevermind":
+                                m 2e "Aww. Well, I hope you feel enough to comfortable to tell me someday."
+                                $ penbool = True
+                            else:
+                                m 1j "That's a lovely pen name!"
+                                m "I think if I saw a pen name like that on a cover, I'd be drawn to it immediately."
+                                $ persistent._mas_penname = penname
+                                $ penbool = True
+                    "I'd rather not; it's embarrassing.":
+                        m 2e "Aww. Well, I hope you feel enough to comfortable to tell me someday."          
+            "No":
+                m 1j "All right!"
+                m "If you ever decide on one, you should tell me!"
+    else:
+        m "Are you still going by [penname], [player]?"
```

Comment:
if you close your session and get this topic the `penname` variable won't be defined at this point so you need to do something like this:
```
        $ penname = persistent._mas_penname
        m "Are you still going by [penname], [player]?"
```